default.nix

```diff
@@ -0,0 +1,81 @@
+# File is taken from nixos/nixpkgs and adapted.
+
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, gfortran
+, nose
+, pytest
+, pytest-xdist
+, numpy
+, pybind11
+, cython
+, meson
+, ninja
+, pythran
+, pkg-config
+, boost175
+}:
+
+let
+
+  excludeIndices = list: indices: let
+    nitems = lib.length list;
+    includeIndices = lib.subtractLists indices (lib.range 0 (nitems - 1));
+  in map (idx: lib.elemAt list idx) includeIndices;
+
+  meson_ = meson.overrideAttrs (oldAttrs: rec {
+    version = "0.59.0.rc2";
+    src = fetchFromGitHub {
+    owner = "mesonbuild";
+    repo = "meson";
+    rev = version;
+    sha256 = "sha256-eC3aUTvUzMS34b2yz67HJ1jADnMiLWgwGsj5/+6njAI=";
+    };
+    patches = excludeIndices oldAttrs.patches [ 2 ]; # remove gir fallback path patch
+  });
+
+
+
+in buildPythonPackage rec {
+  pname = "scipy";
+  version = "dev";
+  format = "other";
+
+  src = ./.;
+
+  checkInputs = [ nose pytest pytest-xdist ];
+  nativeBuildInputs = [ cython gfortran meson_ ninja pythran pkg-config ];
+  buildInputs = [ numpy.blas pybind11 boost175 ];
+  propagatedBuildInputs = [ numpy ];
+
+  # Remove tests because of broken wrapper
+  prePatch = ''
+    rm scipy/linalg/tests/test_lapack.py
+  '';
+
+  doCheck = false;
```

Comment:
test suite currently fails:

```
       >   File "/nix/store/c2rf6xcz15n1l97l5h6w7pqw1b1vxd2k-python3.8-pytest-6.2.4/lib/python3.8/site-packages/_pytest/warnings.py", line 52, in catch_warnings_for_item
       >     apply_warning_filters(config_filters, cmdline_filters)
       >   File "/nix/store/c2rf6xcz15n1l97l5h6w7pqw1b1vxd2k-python3.8-pytest-6.2.4/lib/python3.8/site-packages/_pytest/config/__init__.py", line 1603, in apply_warning_filters
       >     warnings.filterwarnings(*parse_warning_filter(arg, escape=False))
       >   File "/nix/store/c2rf6xcz15n1l97l5h6w7pqw1b1vxd2k-python3.8-pytest-6.2.4/lib/python3.8/site-packages/_pytest/config/__init__.py", line 1579, in parse_warning_filter
       >     category: Type[Warning] = warnings._getcategory(category_)  # type: ignore[attr-defined]
       >   File "/nix/store/6cfajs6lsy9b4wxp3jvyyl1g5x2pjmpr-python3-3.8.9/lib/python3.8/warnings.py", line 262, in _getcategory
       >     raise _OptionError("invalid module name: %r" % (module,)) from None
       > warnings._OptionError: invalid module name: 'scipy._lib._testutils'

```

Reply:
Once I finalize gh-39 (probably this weekend), the whole test suite should pass.

Reply:
This was a dependency problem. It should have disappeared now.